internal_workflow_testsuite.go

```diff
@@ -186,6 +211,38 @@ func newTestWorkflowEnvironmentImpl(s *WorkflowTestSuite) *testWorkflowEnvironme
 	return env
 }
 
+func (env *testWorkflowEnvironmentImpl) newTestWorkflowEnvironmentForChild(options *workflowOptions) *testWorkflowEnvironmentImpl {
+	// create a new test env
+	childEnv := newTestWorkflowEnvironmentImpl(env.testSuite)
+	childEnv.parentEnv = env
```

Comment:
child was added in ExecuteChildWorkflow() method. Put it here also works, will update.